includes/Generator/Order.php

```diff
@@ -104,6 +105,16 @@ public static function generate( $save = true, $assoc_args = array() ) {
 			OrderAttribution::add_order_attribution_meta( $order, $assoc_args );
 		}
 
+		// Set paid and completed dates based on order status.
+		if ( 'completed' === $status || 'processing' === $status ) {
+			// Add random 0 to 36 hours to creation date.
+			$date_paid = date( 'Y-m-d H:i:s', strtotime( $date ) + ( wp_rand( 0, 36 ) * HOUR_IN_SECONDS ) );
+			$order->set_date_paid( $date_paid );
+			if ( 'completed' === $status ) {
+				$order->set_date_completed( $date_paid );
```

Comment:
A minor detail perhaps, but would it make sense to have the completed date potentially offset by another 0 - 36 hours? Seems like for some businesses the "processing" stage includes fulfillment tasks, so it could be a while longer before it moves from processing to completed.